scalafix-core/src/main/scala-3/scalafix/internal/util/PrettyType.scala

```diff
@@ -0,0 +1,732 @@
+package scalafix.internal.util
```

Comment:
These new files in `scala-3` are HUGE - so much duplicated code will be very costly to keep in sync. Instead try to keep the common code in `src/mains/scala` and extract the differences in a smaller file (`Compat.scala`).

Reply:
refactored similarly as Compat, defining different types for different scala versions